### DISCUSSION TITLE
how to set android icon

### DISCUSSION THREAD

**Discussion by xurwxj** [DioxusLabs/dioxus#3448] [General]
os: macos 15.1.1
ver: 0.6.0

[application]

# App (Project) Name
name = "Ftest"
default_platform="android"

[bundle]
icon = ["assets/icon.icns","assets/icon.png"]

[web.app]

# HTML title tag content
title = "findnow"

# include `assets` in web platform
[web.resource]

# Additional CSS style files
style = []

# Additional JavaScript files
script = []

[web.resource.dev]

# Javascript code file
# serve: [dev-server] only
script = []

above setting is invalid.

when build with bundle, will be following:

[dependencies]
dioxus = { version = "0.6.0", features = ["router", "fullstack"] }
when exec: RUST_BACKTRACE=full dx bundle -r --platform android
got following err: 
1.718s ERROR err=UnsupportedFeature("Android bundles are not yet supported")

**Comment by brisk0** [comment]
In case it's of interest still, it appears not to be possible yet, [though there are workarounds](https://github.com/DioxusLabs/dioxus/issues/3685). There is a [pull request](https://github.com/DioxusLabs/dioxus/pull/4063) to make it possible.
